### PR TITLE
Add missing endpoint

### PR DESCRIPTION
**Pull request by shredding**
I added the https://www.stellar.org/developers/horizon/reference/endpoints/trades-for-account.html endpoint. I included a test that i had with this padding removing i had the fix for that as well but it was already done in the meantime.

### CODE REVIEW

File: tests/test_asset.py [hunk 1]
```diff
@@ -38,6 +38,16 @@ def test_xdr(self):
         xdr = b'AAAAAUNOWQAAAAAA01KM3XCt1+LHD7jDTOYpe/HGKSDoQoyL1JbUOc0+E2M='
         assert xdr == self.cny.xdr()
 
+    def test_xdr_padding_will_be_removed(self):
+        xdr = self.cny.to_xdr_object()
+        former_xdr_cny = Asset.from_xdr_object(xdr)
+        assert former_xdr_cny.code == self.cny.code, 'Padding not removed.'
+
+        # check for greater than four as well ...
+        xdr = Asset('BIGGER').to_xdr_object()
```
Comment (overcat) [inline tests/test_asset.py]: Need to provide an asset issuer.

Comment (overcat) [reply tests/test_asset.py]: I think this test is unnecessary.

Comment (shredding) [reply tests/test_asset.py]: Removed it.

**Comment by overcat** [comment]
Thank you.

**Pull request merged**